prefoil/sampling.py

```diff
@@ -280,6 +280,54 @@ def findSpacing(na):
     return s
 
 
+def tanh_distribution(start, end, n, s0, s1):
+    """
+    Hyperbolic tangent distribution based on:
+    https://www.cfd-online.com/Wiki/Structured_mesh_generation
```

Comment:
Could you add today's date in case this link breaks and we need to dig it up from the web archives?